pkg/minishift/openshift/service.go

```diff
@@ -64,41 +104,45 @@ func GetServiceURL(service, namespace string, https bool) (string, error) {
 	cmdName := instanceState.InstanceConfig.OcPath
 	cmdOut, err := runner.Output(cmdName, tokens...)
 	if err != nil {
-		return "", errors.New(fmt.Sprintf("No service with name '%s' defined in namespace '%s'", service, namespace))
+		return "", errors.New(fmt.Sprintf("Service with name '%s' in namespace '%s' doesn't have route associated", service, namespace))
```

Comment:
Would it not be easiest to let _ GetServiceSpec_ call _ GetServiceSpecs_ and then based on the returned  ServiceSpec decide on either the URL to return or the appropriate error message?